webpack.common.config.js

```diff
@@ -128,7 +128,8 @@ var resolve = {
         'diffTool': staticPath('js/diffTool.js'),
         'mathrender': staticPath('js/mathrender.js'),
         'citations': staticPath('js/citations.js'),
-        'tests': staticPath('js/tests')
+        'tests': staticPath('js/tests'),
+        'projectOrganizer': staticPath('js/projectorganizer.js')
```

Comment:
Aha, we are already setting `resolve.root` to `website/static`, which allows us to do `require('js/projectorganizer')`.